clusters/ztp-siteconfig/bevo1.cars2.lab/bevo1-siteconfig.yaml

```diff
@@ -0,0 +1,118 @@
+---
+# yamllint disable rule:line-length
+# yamllint disable rule:comments-indentation
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "bevo1"
+  namespace: "bevo1"
+spec:
+  baseDomain: "cars2.lab"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "img4.12.19-x86-64"
```

Comment:
```suggestion
  clusterImageSetNameRef: "openshift-4.12.19"
```

Reply:
You'll need to update this based on https://github.com/redhat-partner-solutions/vse-carslab-hub/commit/9fd4805ec14bb7d362af8ab33d99a90a3876e0f0.